sunpy/image/transform.py

```diff
@@ -90,35 +90,19 @@ def affine_transform(image, rmatrix, order=3, scale=1.0, image_center=None,
     displacement = np.dot(rmatrix, rot_center)
     shift = image_center - displacement
 
-    # While `use_scipy` is still supported, we have to check which method to actually use
-    method = _get_transform_method(method, use_scipy)
+    method = _get_transform_method(method)
 
     # Transform the image using the appropriate function
     rotated_image = _rotation_registry[method].function(image, rmatrix, shift, order, missing, clip)
 
     return rotated_image
 
 
-def _get_transform_method(method, use_scipy):
+def _get_transform_method(method):
```

Comment:
I didn't really want to remove this function as it was more "work" but as it stands it doesn't do anything that can't be copy and pasted into each method.